pkg/asset/quota/aws/aws.go

```diff
@@ -130,14 +133,26 @@ func others() []quota.Constraint {
 
 func machineTypeToQuota(t string, instanceTypes map[string]InstanceTypeInfo) quota.Constraint {
 	info, ok := instanceTypes[t]
+	warnMessage := "unrecognized instance type specified, skipping vCPU quota check"
```

Comment:
Sorry that I missed this in my last comment. The warning message should use sentences and start with an upper-case letter. Let's also take the opportunity to make it a bit more user-friendly.
```suggestion
	warnMessage := fmt.Sprintf("The instance class is unknown for the instance type %q. The vCPU quota check will be skipped.", t)
```